packages/editor/src/systems/EditorControlSystem.tsx

```diff
@@ -150,9 +151,6 @@ let selectionCounter = 0
 let transformMode: TransformModeType
```

Comment:
we should avoid these module scope variables and grab them from our `EditorHelperState` hyperflux state instead

Reply:
gonna refactor these into the component itself, hopefully most variables will be out of  the editorcontrolsystem